erpnext_com/www/docs/user/manual/en/setting-up/email/email-dropbox.md

```diff
@@ -0,0 +1,27 @@
+<!-- add-breadcrumbs -->
+
+# Email Dropbox
+
+When you receive emails from leads, customers, suppliers and other business partners, you may like to attach that to relevant records in ERPNext. You can save the email and upload it as an attachment. However, this is a cumbersome process. ERPNext offers a better way to handle this.
+
+Each record in ERPNext provides a unique email address to which you can forward emails. ERPNext will attach the email to that record.
+
+## 1. Enable Email Dropbox Feature
+
+In [Email Account](/docs/user/manual/en/setting-up/email/email-account), check 'Enable Email Dropbox'.
+
+<img class="screenshot" alt="Enable Email Dropbox" src="{{docs_base_url}}/assets/img/setup/email/enable_email_dropbox.png">
+
+## 2. Copy Unique Email Address in Any Record
+
+Once the Email Dropbox feature is enables, the unique email address becomes visible in all the records. You can click on the same and the address is copied to your clipboard. You can directly paste it in 'To', 'Cc' or 'Bcc' and send the email.
```

Comment:
spelled it wrong :) should be "is enabled".

Also, how does this sound:


Once the Email Dropbox feature is enabled, the unique email address should be generated for all the records. You can click on the generated email address to copy it to your clipboard, which can then be added to the "To", "Cc", or "Bcc" field of the email that you wish to send.

Reply:
I don't think "should be generated for all the records." is appropriate...